reconcile/utils/vault.py

```diff
@@ -8,6 +8,7 @@
 import hvac
 import requests
 
+from typing import Tuple, Optional, Mapping
```

Comment:
* no need to import `Tuple`, just use `tuple` directly
* `Mapping` should be imported from `collections.abc`

Reply:
Changed!